cmd/genji/shell/command.go

```diff
@@ -136,3 +140,132 @@ func displaySuggestions(in string) error {
 	fmt.Println()
 	return nil
 }
+
+// dumpTable display the content of the given table as SQL statements.
+func dumpTable(db *genji.DB, t *database.Table) error {
```

Comment:
`dumpTable` is always called from within a transaction, we should pass the transaction instead of `db`